I0517 03:53:01.953886 140289360189248 run.py:28] Device: cuda
W0517 03:53:02.276141 140289360189248 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0517 03:53:02.344973 140289360189248 diagnostics.py:438] 5/159 coordinates fall outside of the mask. Removing them.
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
